core/store/src/trie/mod.rs

```diff
@@ -395,21 +396,21 @@ impl RawTrieNodeWithSize {
     }
 }
 
-fn encode_trie_node_with_rc(data: &[u8], rc: u32) -> Vec<u8> {
+fn encode_trie_node_with_rc(data: &[u8], rc: i32) -> Vec<u8> {
```

Comment:
Should the naming reflect that this is now working with delta values instead of absolute values?

Reply:
It's actually both: we write deltas, but read absolute values (aggregation happens in merge operator). Makes sense to do a refactor to move refcount logic from Trie to Store, but we can do it separately